f5_openstack_agent/lbaasv2/drivers/bigip/barbican_cert.py

```diff
@@ -20,12 +20,22 @@
     raise ImportError("Missing Python library barbicanclient."
                       "Install barbicanclient and restart the agent.")
 try:
+    # Try keystoneuath1 first as OpenStack is migrating from keystoneclient
+    # to keystoneauth1. Some systems, particularly liberty, may not
+    # have keystoneauth1 installed. If ImportError, try keystoneclient.
     from keystoneauth1.identity import v2
     from keystoneauth1.identity import v3
     from keystoneauth1.session import Session
+
 except ImportError:
-    raise ImportError("Missing Python library keystoneauth1."
-                      "Install keystoneauth1 and restart the agent.")
+    try:
+        # no keystoneauth1 -- try keystoneclient
+        from keystoneclient.auth.identity import v2
+        from keystoneclient.auth.identity import v3
+        from keystoneclient.session import Session
+    except ImportError:
+        raise ImportError("Missing Python library keystoneclient."
+                          "Install keystoneclient and restart the agent.")
```

Comment:
This import error message is fine for liberty.  But I'd like to see the original message returned for mitaka to install keystoneauth1.  Or in the merge from liberty to mitaka, we retain the original mitaka code (dummy commit).